shared/Microsoft.Azure.Devices.Shared.NetStandard/Microsoft.Azure.Devices.Shared.NetStandard.csproj

```diff
@@ -12,7 +12,9 @@
     <AssemblyName>Microsoft.Azure.Devices.Shared</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <FileAlignment>512</FileAlignment>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors> 
+    <!-- TODO #176: Enable warnings as errors. -->
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors> 
```

Comment:
>false [](start = 27, length = 5)

Why are we turning these off now?

Reply:
This project was missing the FXCop include [below](https://github.com/Azure/azure-iot-sdk-csharp/pull/285/commits/7b962c2b8932f2471f280e667c190e235b6c1dc1#diff-5ec035dad8b7639dca1223004f53893aR84). Adding it triggered a huge amount of warnings. We need to spend time fixing all of those before properly enabling the flag.